README.md

```diff
@@ -145,6 +145,17 @@ if (process.env.http_proxy) {
 }
 ```
 
+### Network retries
+
+Network retry amount can be added with `setMaxNetworkRetries`. This will automatically retry a request `n` amount of times if it failed due to a connection, conflict or rate limiting error.
```

Comment:
`Automatic network retries can be enabled with` *

I think I would move the bit about idempotency keys to this paragraph, with a few edits:

```
Retries use [idempotency keys](...) where appropriate to prevent duplication and exponential backoff with jitter to help in cases of rate limiting.
```

Reply:
Ah, Paul will be out of the office for a week so I'll just commit these changes.